integration-tests/contract-no-annotations-call-by-index-methodsObject.spec.ts

```diff
@@ -3,13 +3,15 @@ import { noAnnotCode, noAnnotInit } from "./data/token_without_annotation";
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
-  describe(`Test contract with no annotations calling methods by index using: ${rpc}`, () => {
-
+  describe(`Test contract origination with no annotations calling methods by index using methodObjects through contract api: ${rpc}`, () => {
+    /** The goal of the test is to verify a contract entrypoint call using the methodsObject method (contract.methodsObject) 
+     *  in case of a contract having no annotation in its code. */
+    
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    it('Test contract with no annotations for methods', async (done) => {
+    it('Verify contract.originate for a contract with no annotations for methods using methodObjects', async (done) => {
```

Comment:
Suggestion: `Verify contract.originate for a contract without annotation and call entry points using methodObjects`

Reply:
changed as suggested

Reply:
@michaelkernaghan looks like the change was not pushed